tests/check_check_log_internal.c

```diff
@@ -72,4 +72,3 @@ Suite *make_log_internal_suite(void)
   
   return s;
 }
-
```

Comment:
I like the extra line at the end of files, though. And, technically all non-empty source files must end in a new line according to the C standard: http://gcc.gnu.org/ml/gcc/2003-11/msg01568.html